shared/postgres/docker-entrypoint-initdb.d/create_table_employees.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS employees;
+DROP SEQUENCE IF EXISTS employees_id_seq;
+CREATE TABLE employees (
+    id int NOT NULL,
+    name varchar(255),
+    full_time bool,
+    updated_at timestamptz,
+    PRIMARY KEY (id)
+);
```

Comment:
Might be good to try out the connector with a more complex data model.

Reply:
If we're giving up on this it's fine, but I'm curious if there's a reason for that?